integrated-scouting-apps/src/public/stylesheets/style.css

```diff
@@ -161,6 +162,60 @@ input[type="checkbox"], .ant-checkbox-inner {
     color: white !important;
     text-align: center !important;
 }
+.pitInput {
+    width: 20%;
+    margin-bottom: 6%;
+    min-width: 35%;
+    height: 25% !important;
+    min-height: 25% !important;
+    border: 4px solid white !important;
+    border-radius: 25px;
+    background-color: #32a7dc;
```

Comment:
change to transparent for all background color

Reply:
done